gatsby-node.js

```diff
@@ -34,12 +34,17 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     reporter.panicOnBuild('Error while running GraphQL query.');
     return;
   }
-
-  result.data.allMarkdownRemark.edges.forEach(({ node }) => {
+  const posts = result.data.allMarkdownRemark.edges;
+  result.data.allMarkdownRemark.edges.forEach(({ node }, index) => {
+    const { path } = node.frontmatter;
     createPage({
-      path: node.frontmatter.path,
+      path,
       component: blogPostTemplate,
-      context: {}, // additional data can be passed via context
+      context: {
+        pathSlug: path,
+        prev: index === 0 ? null : posts[index - 1].node,
+        next: index === (posts.length - 1) ? null : posts[index + 1].node,
+      }, // additional data can be passed via context
```

Comment:
```suggestion
      },
```
let's get rid of this comment now that we use it